Cargo.toml

```diff
@@ -109,7 +108,7 @@ bench = false
 
 [[bin]]
 name = "test_mpc"
-required-features = ["cli", "web-app"]
+required-features = ["cli", "web-app", "test-fixture"]
```

Comment:
one more change is required here - make `ipa-web-client` feature and depend on it here.